packager/react-native-xcode.sh

```diff
@@ -23,11 +23,23 @@ case "$CONFIGURATION" in
     ;;
 esac
 
-# Xcode project file for React Native apps is located in ios/ subfolder
-cd ..
+search_parent_paths_for() {
+  if [ -e "$1" ]; then
+    pwd
+    return 0
+  elif [ "$PWD" != "/" ]; then
+    (cd ..; search_parent_paths_for "$1")
+    return $?
+  fi
+  return 1
+}
 
-set -x
-DEST=$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH
+# Load user environment (which should also handle ndenv or nvm setup)
+[ -e "$HOME/.bashrc" ] && source "$HOME/.bashrc"
```

Comment:
This may cause an issue. For instance, I use Zsh as my shell...but I'm pretty sure my .bashrc file is sitting in my home folder.